Documentation/Authenticators/Authentication-Modules.md

```diff
@@ -86,7 +86,7 @@ GRANT SHOW DATABASES ON *.* TO 'maxscale'@'maxscalehost';
 
 ## Limitations and troubleshooting
 
-When a client logs in to MaxScale, MaxScale sees the client's IP address. When
+When a client logs into MaxScale, MaxScale sees the client's IP address. When
```

Comment:
`into` changes tend to be controversial. happy to drop this or anything else.

Reply:
I'm not a native speaker but to me this seems wrong as the act being described is _log in_ (to a device) not _log_ (fell a tree, write an entry to a journal). The same comment applies to all the _log in to_ changes in this commit.

Reply:
Yes, should be "log in to" and not "log into".

_The proper grammar is "log in to." The phrase "log in" is a verb phrase that means to gain access to a computer system by entering a username and password. The preposition "to" follows the verb "log in" to indicate the thing that you are logging in to, such as a website or a computer program._